source/etiquette/index.md

```diff
@@ -0,0 +1,248 @@
+---
```

Comment:
Same as CSS and JS style guides... I prefer _links_ to sources instead of duplicating content that is expected to evolve over time...

Reply:
👍